.circleci/ci-scripts/ensure-android-sdkmanager.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
This file is a copy of [this](https://github.com/squanchy-dev/squanchy-android/blob/develop/.circleci/ci-scripts/ci-ensure-android-sdkmanager.sh), not sure if we'll need something like this for iOS